src/js/api/auth/login.test.js

```diff
@@ -0,0 +1,41 @@
+import { login } from "./login.js";
+import * as storage from "../../storage/index.js";
+
+// Mock the save function from the storage module
+jest.mock("../../storage/index.js", () => ({
+  save: jest.fn(),
+  load: jest.fn(), // Mock the load function
+}));
+
+// Mock localStorage
+const localStorageMock = (() => {
```

Comment:
Why not refactor this into an import since you are using it in more than one place?